apps/i18n/common/en_us.json

```diff
@@ -2377,6 +2380,9 @@
   "updateUnpluggedLessonProgress": "Update unplugged lesson progress",
   "updateUnpluggedLessonProgressSubHeading": "Make sure your report accurately reflects the unplugged lessons* your class has worked on.",
   "updateUnpluggedProgress": "Update unplugged progress",
+  "upgradeAccountToAssign": "Use a teacher account to assign",
+  "upgradeAccountToAssignButtonText": "Learn how to update account type",
+  "upgradeAccountToAssignHelpText": "You need a teacher account to assign. Please switch to a teacher account or update your account type.",
```

Comment:
Same thing here with "assign." This is a small thing so not a big deal either way, just curious if all 3 of these sentences should be consistent with wording around "assign"?

Reply:
Updated! Thanks